Mage/src/main/java/mage/abilities/effects/common/CreateTokenEffect.java

```diff
@@ -80,6 +90,14 @@ public boolean apply(Game game, Ability source) {
         int value = amount.calculate(game, source, this);
         token.putOntoBattlefield(value, game, source, source.getControllerId(), tapped, attacking);
```

Comment:
See #10786, looks like it can use token with etb ability instead that workaround.

Reply:
Implemented the solution outlined there (see [45a627b](https://github.com/magefree/mage/pull/11865/commits/45a627b8023e4521dfce058c7f2282a771ff59bf)) but this is causing an incorrect interaction with [Chatterfang, Squirrel General](https://scryfall.com/card/mh2/151/chatterfang-squirrel-general), which should create a squirrel which then gains equal +1/+1 counters to the newly-created ooze. In my previous implementation, this happens as expected. With this new implementation with applying an ability to the ooze, the squirrel is created but it does not gain the counters.
![image](https://github.com/magefree/mage/assets/26198472/8d044387-1635-4afa-bd77-15ca3e18541a)

Reply:
ok, I can see the diff. Please, add additional comments to both create token methods about counters workaround with that PR number link like:
`// workaround to add counters to all created tokens, needs for combo support like Chatterfang, Squirrel General with Ochre Jelly or Printlifter Ooze, see #10786`

![IMG_9792](https://github.com/magefree/mage/assets/8344157/450134a0-e738-467b-a87f-082e4e1deec2)

Reply:
Don't forget to put same comments here too:

![IMG_9794](https://github.com/magefree/mage/assets/8344157/6cccc7a6-b09a-441a-a501-94e7fae3087d)

Reply:
Done